src/commands/init/index.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var inquirer = require("inquirer");
+
+
+// inquirer
+//   .prompt([
+//     {
+//       type: "list",
+//       message: "Select the CSS framework which you want to use: ",
+//       name: "framework",
+//       choices: ["Bootstrap", "tailwind", "Materialize"],
+//     },
+//   ])
+//   .then((answers) => {
+//     console.log(answers);
+//   })
+//   .catch((error) => {
+//     if (error.isTtyError) {
+//       console.error(error.message);
+//     } else {
+//       console.error(error.stack);
+//     }
+//   });
```

Comment:
@tonalmathew Do remove the comments

Reply:
```suggestion
// inquirer
//   .prompt([
//     {
//       type: "list",
//       message: "Select the CSS framework which you want to use: ",
//       name: "framework",
//       choices: ["Bootstrap", "tailwind", "Materialize"],
//     },
//   ])
//   .then((answers) => {
//     console.log(answers);
//   })
//   .catch((error) => {
//     if (error.isTtyError) {
//       console.error(error.message);
//     } else {
//       console.error(error.stack);
//     }
//   });
```
@shan-shaji these comments are required for further process

Reply:
@tonalmathew  Ok